bin/server_package/config/ca.crt

```diff
@@ -1,19 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
Why are these removed?

Reply:
these are generated now, leftovers from the old auth model.

Reply:
If it was in a separate commit with an explanatory message, it would cause less confusion :)